qemu/tests/cfg/multi_host.cfg

```diff
@@ -15,7 +15,7 @@
             vms = "vm1"
             start_vm = no
             migration_test_command = help
-            migration_bg_command = "cd /tmp; nohup tcpdump -q -i any -t ip host localhost &"
+            migration_bg_command = "cd /tmp; nohup tcpdump -q -i any -t ip host localhost > tcpdump.out 2>&1 &"
```

Comment:
orig cmd will create ignoring input message, like this
# cd /tmp; nohup tcpdump -q -i any -t ip host localhost &
[1] 28085
# nohup: ignoring input and appending output to ‘nohup.out’
and cannot get the correct output and status, so update it.